searchcore/src/vespa/searchcore/proton/flushengine/flushengine.cpp

```diff
@@ -311,17 +298,17 @@ FlushEngine::flushNextTarget(const vespalib::string & name)
         return "";
     }
     FlushContext::SP ctx = initNextFlush(lst.first);
-    if (ctx.get() == NULL) {
+    if ( ! ctx) {
         LOG(debug, "All targets refused to flush.");
         return "";
     }
     if ( name == ctx->getName()) {
         LOG(info, "The same target %s out of %ld has been asked to flush again. "
                   "This might indicate flush logic flaw so I will wait 1s before doing it.",
                   name.c_str(), lst.first.size());
-        FastOS_Thread::Sleep(1000);
+        std::this_thread::sleep_for(100ms);
```

Comment:
Update printed log message to say "will wait 100ms before doing it" instead?

Reply:
👍 Fixed, thanks.